### ISSUE TITLE
Bump jcabi-parent to latest and cleanup pom

### ISSUE THREAD

**Issue by victornoel** [yegor256/cactoos#1411]
Time for some cleanup in that pom. Let's bump jcabi parent, add some todos to migrate to junit 5 and remove unused stuffs in the pom.

**Comment by victornoel** [comment]
@0crat assign me

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/yegor256/cactoos/issues/1411#issuecomment-653875740))

@victornoel The job [#1411](https://github.com/yegor256/cactoos/issues/1411) assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/c8d184e9-4871-4be1-897b-72fcc3e8dc67); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by victornoel** [comment]
@0crat wait for #1412 to be merged

**Comment by 0crat** [comment]
The architect of the project has changed; @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) is not at this role anymore; @victornoel[/z](https://www.0crat.com/u/victornoel) is the architect now

**Comment by victornoel** [comment]
@0crat assign me

**Comment by victornoel** [comment]
@0crat in

**Comment by victornoel** [comment]
@0crat assign @victornoel

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/yegor256/cactoos/issues/1411#issuecomment-653901862))

@victornoel Job [#1411](https://github.com/yegor256/cactoos/issues/1411) is already in scope

**Comment by 0crat** [comment]
> @0crat assign @victornoel ([here](https://github.com/yegor256/cactoos/issues/1411#issuecomment-653901883))

@victornoel The job [#1411](https://github.com/yegor256/cactoos/issues/1411) assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/484fee46-d79b-4472-a84a-c11ed392afe5); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
> @0crat assign @victornoel ([here](https://github.com/yegor256/cactoos/issues/1411#issuecomment-653901883))

@victornoel There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@d30d334c-b8f5-4e54-8f5c-30e1e363cf4f, thread: PQ-C63314D6Z
com.jcabi.xml.StrictXML[124] java.lang.IllegalArgumentException: 1 error(s) in XML document: -1:-1: Duplicate unique value [gh:yegor256/cactoos#1411] declared for identity constraint of element "agenda".
```

[0.53.15](https://github.com/zerocracy/farm): CID: [98aebee9-fc27-4037-9411-263bbea3dc30](https://www.0crat.com/footprint/C63314D6Z/98aebee9-fc27-4037-9411-263bbea3dc30), Type: "Order was given"

**Comment by 0pdd** [comment]
@victornoel the puzzle [#1416](https://github.com/yegor256/cactoos/issues/1416) is still not solved.

**Comment by 0crat** [comment]
Job was finished in 44 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by victornoel** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/cactoos/issues/1411#issuecomment-654882300))

@victornoel This is what I know about this job in C63314D6Z, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1411](https://github.com/yegor256/cactoos/issues/1411) is not in scope
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Ayegor256%2Fcactoos%231411%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0pdd** [comment]
@victornoel the puzzle [#1453](https://github.com/yegor256/cactoos/issues/1453) is still not solved; solved: [#1416](https://github.com/yegor256/cactoos/issues/1416).

**Comment by 0pdd** [comment]
@victornoel the puzzle [#1586](https://github.com/yegor256/cactoos/issues/1586) is still not solved; solved: [#1416](https://github.com/yegor256/cactoos/issues/1416), [#1453](https://github.com/yegor256/cactoos/issues/1453).

**Comment by 0pdd** [comment]
@victornoel the puzzle [#1714](https://github.com/yegor256/cactoos/issues/1714) is still not solved; solved: [#1416](https://github.com/yegor256/cactoos/issues/1416), [#1453](https://github.com/yegor256/cactoos/issues/1453), [#1586](https://github.com/yegor256/cactoos/issues/1586).
